src/Xamarin.Android.Build.Tasks/Tasks/LinkApplicationSharedLibraries.cs

```diff
@@ -0,0 +1,266 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using ThreadingTasks = System.Threading.Tasks;
```

Comment:
This could use the new [AsyncTaskExtensions](https://github.com/xamarin/xamarin-android/blob/0d211ceb5c21f9f47d7894c1400596052055fb0c/src/Xamarin.Android.Build.Tasks/Utilities/AsyncTaskExtensions.cs) we have now. It makes things much nicer!

The idea is you shouldn't have to do a weird using like this: `using ThreadingTasks = System.Threading.Tasks;`